docs/source/commands/license.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: The Rover license command
+description: Manage Enterprise licenses 
+---
+
+Rover commands that manage Enterprise licenses for Apollo begin with `rover license`.
```

Comment:
Other than fetching offline licenses, there isn't any other use for the `license fetch` command, right?

As a reader, I might see this and wonder if I'm supposed to be using these for license management, so it might be good to add a note that regular (i.e. non-offline) licenses are automatically fetched via Uplink, so there's no need to run this command.